Main_Modelling_file.ipynb

```diff
@@ -4,160 +4,14 @@
    "cell_type": "markdown",
```

Comment:
<p>Needs to be a Markdown instead of python code.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Berga20/DS-Project-Group4/pull/55/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>